src/main/java/com/minecolonies/api/blocks/decorative/AbstractBlockGate.java

```diff
@@ -83,6 +83,16 @@ public AbstractBlockGate(final String name, final float hardness, final int maxW
         this.hardness = hardness;
     }
 
+    public int getMaxWidth()
```

Comment:
no javadoc?

Reply:
I thought about it, but getters seemed sufficiently trivial.